build.sbt

```diff
@@ -8,31 +8,43 @@ import uk.gov.hmrc.versioning.SbtGitVersioning
 
 lazy val appName = "api-platform-test-user"
 lazy val appDependencies: Seq[ModuleID] = compile ++ test
+lazy val akkaVersion = "2.5.23"
+lazy val akkaHttpVersion = "10.0.15"
 
 lazy val compile = Seq(
-  "uk.gov.hmrc" %% "bootstrap-play-25" % "5.1.0",
-  "uk.gov.hmrc" %% "play-ui" % "8.3.0-play-25",
-  "uk.gov.hmrc" %% "play-hmrc-api" % "3.4.0-play-25",
+  "uk.gov.hmrc" %% "bootstrap-play-26" % "1.4.0",
+  "uk.gov.hmrc" %% "play-ui" % "8.8.0-play-26",
+  "uk.gov.hmrc" %% "play-hmrc-api" % "4.1.0-play-26",
   "uk.gov.hmrc" %% "play-json-union-formatter" % "1.5.0",
-  "uk.gov.hmrc" %% "domain" % "5.6.0-play-25",
-  "uk.gov.hmrc" %% "mongo-lock" % "6.15.0-play-25",
-  "org.scalacheck" %% "scalacheck" % "1.13.5",
-  "org.mindrot" % "jbcrypt" % "0.4"
+  "uk.gov.hmrc" %% "domain" % "5.6.0-play-26",
+  "uk.gov.hmrc" %% "mongo-lock" % "6.15.0-play-26",
+
+  "org.mindrot" % "jbcrypt" % "0.4",
+
+  "com.typesafe.play" %% "play-json" % "2.6.14",
+  "com.typesafe.play" %% "play-json-joda" % "2.6.4",
+
+  "com.typesafe.akka" %% "akka-stream"    % akkaVersion     force(),
+  "com.typesafe.akka" %% "akka-protobuf"  % akkaVersion     force(),
+  "com.typesafe.akka" %% "akka-slf4j"     % akkaVersion     force(),
+  "com.typesafe.akka" %% "akka-actor"     % akkaVersion     force(),
+  "com.typesafe.akka" %% "akka-http-core" % akkaHttpVersion force()
 )
 
 lazy val scope: String = "test, it"
 
 lazy val test = Seq(
-  "uk.gov.hmrc" %% "hmrctest" % "3.9.0-play-25" % scope,
-  "uk.gov.hmrc" %% "reactivemongo-test" % "4.15.0-play-25" % scope,
+  "uk.gov.hmrc" %% "hmrctest" % "3.9.0-play-26" % scope,
+  "uk.gov.hmrc" %% "reactivemongo-test" % "4.15.0-play-26" % scope,
   "org.scalatest" %% "scalatest" % "3.0.4" % scope,
-  "org.scalatestplus.play" %% "scalatestplus-play" % "2.0.0" % scope,
+  "org.scalatestplus.play" %% "scalatestplus-play" % "3.1.2" % scope,
   "org.pegdown" % "pegdown" % "1.6.0" % scope,
   "com.typesafe.play" %% "play-test" % PlayVersion.current % scope,
```

Comment:
I think this is transitive dep of hmrc test

Reply:
Nah